CMakeLists.txt

```diff
@@ -50,7 +50,11 @@ option(BUILD_CV_KINFU_EXAMPLE "Build OpenCV KinectFusion example" OFF)
 add_subdirectory(wrappers)
 
 if (BUILD_EXAMPLES AND BUILD_GRAPHICAL_EXAMPLES)
-    add_subdirectory(third-party/glfw)
+    find_package(glfw3 QUIET)
```

Comment:
Hi @radfordi
After #3051 we depend on GLFW 3.3 and not just 3+. Moreover, this version is missing from standard apt repos.
I'm concerned this would break compilation when apt version of glfw is installed. 
Can we find packaged based on minor as well?

Reply:
Thanks for the explanation.  Fixed.